project.toml

```diff
@@ -3,4 +3,32 @@ requires = [
     "setuptools>=42",
     "wheel"
 ]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ivy"
+version = "{__version__}"
+description = "The unified machine learning framework, enabling framework-agnostic functions, layers and libraries."
+readme = "README.rst"
```

Comment:
Readme is now `README.md`, we also run custom code on the raw document to remove dark mode blocks 

Ref: https://github.com/unifyai/ivy/blob/master/setup.py#L28-L45